helm/idol-nifi/resources/nifi-postStart.sh

```diff
@@ -12,8 +12,19 @@
 # END COPYRIGHT NOTICE
 
 logfile=/opt/nifi/nifi-current/logs/post-start.log
-/scripts/security.sh | tee -a ${logfile}
-/scripts/wait.sh | tee -a ${logfile}
-/scripts/connect-registry.sh | tee -a ${logfile}
-/scripts/import-flow.sh | tee -a ${logfile}
-echo [$(date)] postStart completed | tee -a ${logfile}
\ No newline at end of file
+(
+    /scripts/security.sh
+
+    grep nifi-0. /etc/hostname
```

Comment:
This should similarly get updated to use determine correct value using POD_NAME
```suggestion
    statefulsetname=${POD_NAME%-*}
    grep "${statefulsetname}-0." /etc/hostname
```